### PR TITLE
Add sorting functionality to flights list

### PR DESCRIPTION
**Pull request by mzanrosso**
Reference: https://github.com/moshobo/appintheair-exporter/issues/7

### CODE REVIEW

File: appintheair_exporter.py [hunk 1]
```diff
@@ -160,6 +160,8 @@ def main():
     if args.filename:
         filename = args.filename
 
+    flights = sorted(flights, key=lambda d: d['departure_timestamp_gmt'])
```
Comment (moshobo) [inline appintheair_exporter.py:163]: Can you add `reverse=True` so that the most recent flights are at the top?

**Comment by moshobo** [comment]
Thanks for working on this issue @mzanrosso!

**Comment by moshobo** [comment]
Hey @mzanrosso there are a few conflicting changes now. I wanted to see if you were still planning on completing this PR?

**Comment by mzanrosso** [comment]
> Hey @mzanrosso there are a few conflicting changes now. I wanted to see if you were still planning on completing this PR?

Hey I'm sorry, I don't currently have time for implementing a better solution.

**Comment by moshobo** [comment]
No worries, @mzanrosso. Since your branch was behind by a few commits, I've gone ahead and submitted a slightly modified version of your change in #15 . I will close this PR now that the change is submitted.

Thanks again for working on this!

**Pull request closed without merging**